React/main/src/app/screens/Profile/components/ProfileCard/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Image from '~components/Image';
+
+import TextHolder from '~components/TextHolder';
+
+import style from './styles.scss';
+
+const ProfileCard = ({ data }) => (
+  <div className={style['profile-card']}>
+    <div className={style['profile-pic-container']}>
+      <Image src="http://google.com" className={style['profile-pic']} alt="Profile Pic" />
```

Comment:
Use de API and show an image per user.if the user not has a picture, show a default